### ISSUE TITLE
Error: Cannot find module 'ampersand-collection'

### ISSUE THREAD

**Issue by sean-shirazi** [ciscospark/spark-js-sdk#2]
node v6.1.0
npm v3.8.6
on OS X and AWS Elastic Beanstalk Linux

Just updated to 0.6.6 from 0.6.4 and I get this error
```
module.js:442
    throw err;
    ^

Error: Cannot find module 'ampersand-collection'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/var/app/current/node_modules/@ciscospark/plugin-wdm/dist/device/feature-collection.js:7:28)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

**Comment by ianwremmel** [comment]
Thanks for the report @sean-shirazi! Looks like I missed a dependency because of npm3. I'll have a release out shortly with a fix.

**Comment by ianwremmel** [comment]
@sean-shirazi, I just published 0.6.8. Please let me know if you still see issues.
